### PR TITLE
[FIX] Admin page crashing when commit hash is null

### PR DESCRIPTION
**Pull request by cprice-kgi**
## Proposed changes (including videos or screenshots)

If the commit hash happens to be null, the administration page will still attempt to slice the value and display it. This causes the admin page to not display, and essentially crash the web app. This fixes it by checking for a null value first.

## Issue(s)

https://github.com/RocketChat/Rocket.Chat/issues/22031

## Steps to test or reproduce

Attempt to access the admin page in an environment where the deployed code does not have a commit hash value. Apparently this is the case when using the zip archive from the github releases page, but certainly there could be other scenarios that replicate a null commit hash.

### CODE REVIEW

**Review by ggazzo — APPROVED** [review]

File: client/views/admin/info/DeploymentCard.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const DeploymentCard = memo(function DeploymentCard({ info, statistics, instance
 					</Card.Col.Section>
 					<Card.Col.Section>
 						<Card.Col.Title>{t('Commit_details')}</Card.Col.Title>
-						{t('HEAD')}: ({s(() => commit.hash.slice(0, 9))}) <br />
+						{t('HEAD')}: ({s(() => commit.hash ? commit.hash.slice(0, 9) : '')}) <br />
```
Comment (ggazzo) [inline client/views/admin/info/DeploymentCard.js:69]:
```suggestion
						{t('HEAD')}: ({s(() => (commit.hash ? commit.hash.slice(0, 9) : ''))}) <br />
```

**Review by ggazzo — APPROVED** [review]

**Comment by ankar84** [comment]
I have similar issue with node version on my test deployment
`TypeError: Cannot read property 'nodeVersion' of undefined` in DevTools and crash

**Comment by dougfabris** [comment]
can you create an issue with more information about it, please @ankar84?

**Comment by ankar84** [comment]
> can you create an issue with more information about it, please @ankar84?

Unfortunately yesterday we restore db from production deployment to that test deployment and now there is not that problem there - info page opens normally. 
It was strange, because previously we did the same - restore prod to that test and in some moment we saw that bug on test but on prod was all fine. Docker images same, db almost same.

**Pull request merged**